src/components/Articles/Articles.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-card flat id="articles">
+    <confirm-dialog ref="confirm" attach="#app" />
```

Comment:
It looks like you don't use this dialog in this component. Please remove accordingly.

Reply:
Removed. Thanks!